tasks/src/sentence-similarity/about.md

```diff
@@ -0,0 +1,80 @@
+## Use Cases
+You can query through documents to extract information using sentence similarity models. First you can rank the documents themselves with Passage Ranking models, get the best performing document and search in it with sentence similarity models, pick the sentence with best cosine similarity. 
```

Comment:
I'm not sure if "query through documents" is super clear. Maybe this could be worded a bit differently